src/main/webapp/src/components/entityAnalysisInfo.js

```diff
@@ -81,23 +81,45 @@ class EntityAnalysisInfo extends React.Component {
       return <p>No entities were found.</p>;
     }
 
+    /** Generate a custom tooltip for each entity */
+    function getTooltip(entity) {
+      return '<div style="font-size:15px">' + entity.name + '<br>' 
+          + '<b>Type: </b>' + entity.type + '<br>' 
+          + '<b>Salience: </b>' + entity.salience + '<br>' 
+          + ((entity.wikiLink !== "") ? 'Click for wiki link' : '')
+          + '</div>';
+    }
+
     /** Gather the most important words and their data in simple arrays. */
     var top10WordsData = [];
```

Comment:
the `top10WordsData` does not look simple to me. We add things like `Word and Type`, but also a subarray with the entity name and also type salience. Also, in the line below, we add `{ role: "tooltip", type: "string", 'p': {'html': true} }`, which is even less clear to me. Can you please add some documentation here explaining what this array should represent? Also, it might be worth to extract some things to variables with a meaningful name, such as the object above (`'role: "tooltip", type...`)

Reply:
Hmm yea, it used to be simpler in the old days.